common/src/main/java/io/github/miniplaceholders/common/PlaceholdersCommand.java

```diff
@@ -85,34 +89,34 @@ private Command.Builder<A> rootBuilder() {
     }
 
     public void register() {
-        new MinecraftExceptionHandler<A>()
-                .withDefaultHandlers()
-                .withDecorator(component -> Component.text().append(TITLE).appendSpace().append(component).build())
-                .apply(commandManager, AudienceProvider.nativeAudience());
+        MinecraftExceptionHandler.create(AudienceProvider.nativeAudience())
+                .defaultHandlers()
+                .decorator(component -> Component.text().append(TITLE).appendSpace().append(component).build());
```

Comment:
Only thing missed here was registering the exception handler back to the command manager. Should be #registerTo(CommandManager) at the end of the chain there. Might need to play with generics on the create call to let it accept the manager.